javatests/arcs/core/host/AbstractArcHostTest.kt

```diff
@@ -177,7 +178,7 @@ open class AbstractArcHostTest {
         host.getFooHandle().dispatchStore(entity)
         var storedEntity = EntityBase("EntityBase", DummyEntity.SCHEMA)
         storedEntity.setSingletonValue("text", "Watson")
-        assertThat((host.getFooHandle() as ReadWriteSingletonHandle<EntityBase>).fetch()?.equals(
+        assertThat((host.getFooHandle() as ReadWriteSingletonHandle<EntityBase>).dispatchFetch()?.equals(
```

Comment:
While you're touching this, can you fix a bug? The assert is bracketed incorrectly so it's not actually testing anything...

```
assertThat((host.getFooHandle() as ReadWriteSingletonHandle<EntityBase>).dispatchFetch()).equals(storedEntity)
```

Reply:
No problem